plugins/GSdx/GS.h

```diff
@@ -1425,3 +1425,25 @@ enum class Filtering : uint8
 	Trilinear_Bilinear_Forced,
 	Trilinear_Always
 };
+
+//Enum order matters! Don't change it!
+enum class CRCHackLevel : int8
+{
+	Automatic = -1,
+	None = 0,
+	Minimum = 1,
+	Partial = 2,
+	Full = 3,
+	Aggressive = 4
```

Comment:
Huh... well, ideally I think all the values should be removed, but the ini should use strings instead of numbers to represent crc levels. Of course that'd completely break ini compatibility. You may want opinion from someone other than me. ;)